clients/src/main/resources/common/message/AddPartitionsToTxnRequest.json

```diff
@@ -23,17 +23,35 @@
   // Version 2 adds the support for new error code PRODUCER_FENCED.
   //
   // Version 3 enables flexible versions.
-  "validVersions": "0-3",
+  //
+  // Version 4 adds VerifyOnly field to check if partitions are already in transaction and adds support to batch multiple transactions.
+  "validVersions": "0-4",
```

Comment:
Would it make sense to set `latestVersionUnstable` so that we reserve some flexibility to change the API after we merge this?